src/foraging_gui/rigcontrol.py

```diff
@@ -14,6 +14,7 @@ def msg_handler(self, address, *args):
         msg = OSCMessage(address, args)
         CurrentMessage=[msg.address,args[1][0],msg.values()[2],msg.values()[3]]
         self.msgs.put([msg,args])
+        print(CurrentMessage)
```

Comment:
Why print the OSC message to the terminal?

Reply:
It's nice to see an immediate response when debugging. For example, debugging a lick detector.